api/graphql/makeModels.js

```diff
@@ -198,10 +201,7 @@ export default function makeModels (userId, isAdmin) {
             sort: sortBy,
             topic
           })
-      },
-      filter: nonAdminFilter(relation => relation.query(q => {
```

Comment:
Solved this as you recommend but needed to wrap these in an add'l `.where(q => ...` (AND) to get the right results. Works fine.